apps/avatax/e2e/data/functions/index.ts

```diff
@@ -0,0 +1 @@
+import "./user";
```

Comment:
This was the previous comment:

> I think instead integration folder we should have e.g. test, because there is a blurry line between these tests and we don't want to figure out every time "is this integration or e2e?"

I don't think `test` or `integration` were good names, because it wasn't clear - is it a end to end test? is it just making requests? does it use requests recordings? UI test? I've changed folder name from `integration` → `e2e`

We already have unit tests in this app along the app's code with `.test.ts`, these files have `.spec.ts` suffix instead.

Reply:
ok non blocking

Reply:
IMHO it's better to make clear distinction between integration / e2e (whatever we name them) and unit tests in the app